examples/with-cookie-auth/utils/auth.js

```diff
@@ -74,7 +74,7 @@ function auth (ctx) {
   }
 
   // We already checked for server. This should only happen on client.
-  if (!token) {
+  if (!ctx.req && !token) {
```

Comment:
It sounds more like you're passing something into the `auth` function that does not contain `req` 🤔

Reply:
Because it shouldn't end up passing `if (ctx.req && !token) {`